files/cluster_provision.yml

```diff
@@ -57,23 +58,34 @@
                                 preseed_type="kickstart"
                                 kernel_desc="CentOS 7.5"
                                 initrd_desc="CentOS 7.5"
-                                preseed_name="CentOS Upstream"
+                                preseed_name="CentOS.Upstream"
```

Comment:
Why renaming with a dot?

Reply:
Strange, but in any case the name has the dot !

Reply:
I don't care what it's called, just wondering why the change of heart. :)

I'm happy with or without dots.